### DISCUSSION TITLE
Getting Started Outdated ?

### DISCUSSION THREAD

**Discussion by chipbrommer** [lepoco/wpfui#467] [Q&A]
Hello, 

I am completely new to WPF. I'm attempt to get started with using the WPF-UI extension but seem to be hitting road blocks even with the "Getting Started " section. I am getting errors that state: 
`
error MC3074: The tag 'SymbolIcon' does not exist in XML namespace 'http://schemas.lepo.co/wpfui/2022/xaml'. Line 11 Position 10.
`

code: 
```
<Window x:Class="TestUi.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:ui="http://schemas.lepo.co/wpfui/2022/xaml"
        xmlns:local="clr-namespace:TestUi"
        mc:Ignorable="d"
        Title="MainWindow" Height="450" Width="800">
    <Grid>
        <ui:SymbolIcon Symbol="Fluent24" />
    </Grid>
</Window>
```

I'm seeing the same issues for the edits to the `App.xaml` file. Stating: 
`error MC3074: The tag 'ThemesDictionary' does not exist in XML namespace 'http://schemas.lepo.co/wpfui/2022/xaml'. Line 10 Position 18.`

Is the content just outdated?

**Comment by chucker** [comment]
Do you have WpfUi installed via NuGet? Make sure you build once after installing the package. The XAML editor is a bit unreliable when references have been changed. That would be my guess — you installed the package but VS hasn't built yet, so it doesn't really know WpfUi yet.

Also make sure your `MainWindow.xaml` (assuming that's its name) also has a corresponding `MainWindow.xaml.cs`, which should probably look something like:

```csharp
using System.Windows;

namespace TestUi
{
    /// <summary>
    /// Interaction logic for MainWindow.xaml
    /// </summary>
    public partial class MainWindow : Window
    {
        public MainWindow()
        {
            InitializeComponent();
        }
    }
}
```

Importantly, the namespaces must match.
